sdk/communication/communication-phone-numbers/rollup.base.config.js

```diff
@@ -41,7 +41,7 @@ export function nodeConfig(test = false) {
 
   if (test) {
     // Entry points - test files under the `test` folder(common for both browser and node), node specific test files
-    baseConfig.input = ["dist-esm/test/*.spec.js", "dist-esm/test/node/*.spec.js"];
+    baseConfig.input = ["dist-esm/**/*.spec.js"];
```

Comment:
i had a comment like this on my PR, this is supposed to just be for node, so please update the pattern accordingly